### PR TITLE
OCPBUGS#23165: Moved user-defined tags section after installation configuration

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.11+

Issue: [OCPBUGS-23165](https://issues.redhat.com/browse/OCPBUGS-23165)

Link to docs preview: 
GCP: https://70396--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations
Azure: https://70396--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations

QE review:
- [x] QE has approved this change. @jianli-wei

### CODE REVIEW

File: installing/installing_azure/installing-azure-customizations.adoc [hunk 1]
```diff
@@ -61,6 +57,10 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 * For more details about Accelerated Networking, see xref:../../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-accelerated-networking_creating-machineset-azure[Accelerated Networking for Microsoft Azure VMs].
 
+include::modules/installation-user-defined-tags-azure.adoc[leveloffset=+1]
```
Comment (mramendi) [inline installing/installing_azure/installing-azure-customizations.adoc:60]: Maybe consider removing "the" from the heading here, to make it align with the next heading? Your call, though.

**Review by mramendi — COMMENTED** [review]
I made one minor comment, otherwise lgtm. As I am in the process of induction into the peer review squad, @JoeAldinger will finish this review.

File: installing/installing_azure/installing-azure-customizations.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply installing/installing_azure/installing-azure-customizations.adoc:60]: Good suggestion. Though this isn't in scope of this change, I will fix it.

Comment (snarayan-redhat) [reply installing/installing_azure/installing-azure-customizations.adoc:60]: is it ok to be marked as peer-review-done?

**Review by JoeAldinger — APPROVED** [review]
reorg lgtm

**Pull request merged**